components/Progressbar/Circular/CircularProgressbar.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="circular" :style="{width,height}"></div>
+</template>
+
+<script>
+export default {
+  name: 'CircularProgressbar',
+  props: {
+    width: { type: String, default: '50px' },
```

Comment:
maybe this can be just `size` instead of `width` and `height`.

Reply:
Fixed.